expression/builtin_op_vec.go

```diff
@@ -91,11 +91,36 @@ func (b *builtinLogicOrSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column)
 }
 
 func (b *builtinBitOrSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinBitOrSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+
+	numRows := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETInt, numRows)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[1].VecEvalInt(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	arg0s := result.Int64s()
+	arg1s := buf.Int64s()
+
+	result.MergeNulls(buf)
+
+	for i := 0; i < numRows; i++ {
+		if result.IsNull(i) {
+			continue
+		}
```

Comment:
We can remove this check here, nothings changes but be more friendly to the CPU branch prediction.

Reply:
That makes a lot of sense, pushed in a commit to incorporate this